core/server/common/src/main/java/alluxio/master/journal/JournalContext.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.journal;
+
+import alluxio.proto.journal.Journal;
+
+import java.io.Closeable;
+
+/**
+ * Context for storing journaling information.
+ */
+public interface JournalContext extends Closeable {
+  /**
+   * @param entry the {@link Journal.JournalEntry} to append to the journal
+   */
+  void append(Journal.JournalEntry entry);
```

Comment:
nit: this can be just `append(JournalEntry entry)` if you import `alluxio.proto.journal.Journal.JournalEntry`